server/ladder_service.py

```diff
@@ -30,12 +32,18 @@ def __init__(self, database: FAFDatabase, games_service: GameService):
 
         # Hardcoded here until it needs to be dynamic
         self.queues = {
-            'ladder1v1': MatchmakerQueue('ladder1v1', games_service)
+            'ladder1v1': MatchmakerQueue('ladder1v1', game_service=games_service),
+            'ladder2v2': MatchmakerQueue(
+                'ladder2v2',
```

Comment:
please do not treat the queues as ladder or represent them to users as such in any way. Why not just the 2v2 queue?